tests/atest/rules/duplications/duplicated-variables-import/expected_output.txt

```diff
@@ -1,2 +1,8 @@
 ${rules_dir}${\}test.robot:3:1 [W] 0807 Duplicated variables import with path "variables.py" in suite
-${rules_dir}${\}test.robot:5:1 [W] 0807 Duplicated variables import with path "variables.py" in suite
\ No newline at end of file
+${rules_dir}${\}test.robot:5:1 [W] 0807 Duplicated variables import with path "variables.py" in suite
+${rules_dir}${\}test.robot:9:1 [W] 0807 Duplicated variables import with path "other1.py" in suite
+${rules_dir}${\}test.robot:10:1 [W] 0807 Duplicated variables import with path "other1.py" in suite
+${rules_dir}${\}test.robot:12:1 [W] 0807 Duplicated variables import with path "vars2.yaml" in suite
+${rules_dir}${\}test.robot:13:1 [W] 0807 Duplicated variables import with path "vars2.yaml" in suite
```

Comment:
I wouldn't like to have this message duplicated for each line. If we have 2 exactly the same lines, only the latter should be reported.

Reply:
But what if you have it thrice? I know its repeating but there is benefit of easily seeing where the duplicate are. Unless we want to modify issue message to something along "variable import with path x already in x line"

Reply:
I think that reporting each duplicate twice (or more) will lead to unnecessarily big amount of artificial issues. The issue here is only one - duplication, not two. I will opt for having one reported message with an indicator of where the original one is, as you proposed.